index.html

```diff
@@ -21,27 +21,218 @@
+                        </div>
+                        <div>
+                            <button class="btn">See Project</button>
+                        </div>
+                    </div>
+                </div>
+                
+                
+                
+                
+                
+            </div>
+        </section>
+        <section>
+            <div class="flex-container">
+                <div id="about-title">
+                    <div id="about-box" class="font-normal">
+                        <p>About me</p>
+                    </div>
+                    <div id="about-text" class="font-normal">
+                        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                            don’t hesitate to contact me.</p>
+                    </div>
+                    <div class="btn-div">
+                        <button class="btn">Get my resume</button>
+                    </div>
+                </div>
+                <div class="about-items">
+                    <div class="frame-img">
+                        <img src="images/icon-languages.svg">
+                    </div>
+                    <div class="frame-card">
+                        <h4>Languages</h4>
+                    </div>
+                    <div class="frame-list">
+                        <ul>
+                            <li>JavaScript</li>
+                            <li>Ruby</li>
+                            <li>HTML</li>
+                            <li>CSS</li>
+                        </ul>
+                    </div>
+                </div>
+                <div class="about-items">
+                    <div class="frame-img">
+                    <img src="images/icon-frameworks.svg">
+                </div>
+                <div class="frame-card">
+                    <h4>Frameworks</h4>
+                </div>
+                <div class="frame-list">
+                    <ul>
+                        <li>React.js</li>
+                        <li>Ruby on rails</li>
+                        <li>RSpec</li>
+                        <li>Capybara</li>
+                        <li>Selenium</li>
+                    </ul>
+                </div>
+            </div>
+                <div class="about-items">
+                    <div class="frame-img">
+                        <img src="images/icon-skillsim.svg">
+                    </div>
+                <div class="frame-card">
+                    <h4>Skills</h4>
+                </div>
+                <div class="frame-list">
+                    <ul>
+                        <li>Database Management</li>
+                        <li>Version Control</li>
+                        <li>CLI</li>
+                        <li>Web Development</li>
+                        <li>API Design</li>
+                    </ul>
+                </div>
```

Comment:
- As I show in the picture below the link to the image is broken kindly fix that and reduce  the size of the button and adjust abit the disposition it will make your design stick more to the Figma design :+1: 

## Your design 

![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/392bd773-7d35-4868-8ce3-40acbbe5f818)




## Figma design 

![image](https://github.com/Armynerh/personal_portfolio/assets/95297251/9f7ddc85-6059-43a8-9dbb-3a70bcfdb3ea)